app/Filament/Server/Pages/Settings.php

```diff
@@ -75,6 +77,20 @@ public function form(Form $form): Form
                                     ->autosize()
                                     ->live(onBlur: true)
                                     ->afterStateUpdated(fn ($state, Server $server) => $this->updateDescription($state ?? '', $server)),
+                                TextInput::make('avatar_url')
+                                    ->label('Server Avatar')
+                                    ->disabled(fn () => !auth()->user()->can(Permission::ACTION_SETTINGS_AVATAR, $server))
+                                    ->columnSpan([
+                                        'default' => 1,
+                                        'sm' => 2,
+                                        'md' => 4,
+                                        'lg' => 6,
+                                    ])
+                                    ->formatStateUsing(fn ($state) => $state ?? (new UiAvatarsProvider())->get($this->getRecord()))
+                                    ->url()
+                                    ->live(onBlur: true)
+                                    ->afterStateUpdated(fn ($state, Server $server) => $this->updateAvatar($state ?? '', $server))
+                                    ->prefix(fn ($state) => new HtmlString("<img src='$state' width='24vh' height='24vh' class='fi-avatar object-cover object-center rounded-md h-8 w-8 fi-tenant-avatar shrink-0'/>")),
```

Comment:
Use [`<x-filament::avatar>`](https://filamentphp.com/docs/3.x/support/blade-components/avatar) instead of `<img>`.